tests/test_runner.py

```diff
@@ -247,7 +249,19 @@ def test_runner_run_ok_without_debug(self):
 
         output = f.getvalue()
         self.assertIn(f"Checking {nasl_file}", output)
-        self.assertNotIn("Running plugin check_missing_desc_exit", output)
+        self.assertNotIn("Results for plugin check_missing_desc_exit", output)
```

Comment:
Why? The output is always 
```
ℹ Checking common2/2008/gb_w3c_amaya_mult_bof_vuln_dec08_lin.nasl (413/415)
ℹ     Running plugin check_openvas_lint
×         openvas-nasl-lint not found within your PATH. Please install 'openvas' and make sure it is available within your PATH.
ℹ     Running plugin check_updated_date_version
×         Changed VT has a not updated script_version();
```
Or not?

```suggestion
        self.assertNotIn("Running plugin check_missing_desc_exit", output)
```

Reply:
Because actually the plugin has run already and therefore the statement is technically wrong

Reply:
Yes that is correct. But it is not enough to change this in a single assertion within a test. :P 
Should we change it all over, or leave it?

Reply:
Oh sorry this change should not have been included here. I'll fix that.